### ISSUE TITLE
"kubectl expose" output does not have IP

### ISSUE THREAD

**Issue by thockin** [kubernetes/kubernetes#10836]
```
$ kubectl expose rc hostnames --port=80 --target-port=9376
NAME        LABELS          SELECTOR        IP(S)     PORT(S)
hostnames   app=hostnames   app=hostnames             80/TCP

$ kubectl get svc hostnames
NAME        LABELS          SELECTOR        IP(S)        PORT(S)
hostnames   app=hostnames   app=hostnames   10.0.1.175   80/TCP
```

The output of the expose command should include the cluster IP or at least say <not yet assigned> or something.  But IPs are assigned synchronously for now, so it SHOULD be available.

**Comment by Kielan** [comment]
I've run into this issue multiple times. Does accessing the IP of the svc of the same name work?

**Comment by bgrant0607** [comment]
kubectl expose should not dump the resource. See https://github.com/kubernetes/kubernetes/issues/5906#issuecomment-117832140 and #12845.

cc @hurf

**Comment by hurf** [comment]
@bgrant0607 - again `rc hostname exposed`?

**Comment by hurf** [comment]
It seems only local generated object is printed out, I'll print the created object in the fix.

**Comment by janetkuo** [comment]
Closing this since it's fixed by #13187
